exercises/practice/yacht/.meta/example.c

```diff
@@ -0,0 +1,108 @@
+#include "yacht.h"
+
+#include <assert.h>
+#include <stdbool.h>
```

Comment:
```suggestion
```

I think this is unused

Reply:
A leftover from when `has_count` returned a `bool`. Removed that now.